docs/checkouts/hosted-checkout.mdx

```diff
@@ -0,0 +1,18 @@
+# Hosted Checkout
+
+Hosted Checkout is the quickest and easiest way to use JustiFi checkouts without the need to build a user interface. Once a checkout is created via API, a link to that checkout can be shared and will take the user to a JustiFi hosted checkout solution.
+
+**Note:** JustiFi checkouts expire after 1 week. Please keep this in mind when generating and sharing links to hosted checkouts.
+
+## Getting Started
+
+First, create a checkout following our [Create a Checkout API documentation](https://docs.justifi.tech/api-spec#tag/Checkouts/operation/CreateCheckout).
+
+Upon successful checkout creation, use the checkout ID from the response to create a hosted checkout link:
+
+```components.justifi.ai/checkout/:checkoutId```
+
+## Custom Redirects
+
+JustiFi hosted checkout has default behavior and redirects for when a checkout is completed successfully or has expired. If the desired behavior is to redirect back to your application upon completion of the checkout, you can use the `success_redirect_url` parameter to override the default redirect:
```

Comment:
Can you please update the first sentence to:
By default JustiFi Hosted Checkout redirects to a generic `Checkout Complete` page after successful checkout or a `Checkout Session Expired` page if the checkout has expired.